BeeSwift/GalleryViewController.swift

```diff
@@ -459,7 +459,27 @@ class GalleryViewController: UIViewController, UICollectionViewDelegateFlowLayou
     }
     
     func collectionView(_ collectionView: UICollectionView, layout collectionViewLayout: UICollectionViewLayout, sizeForItemAt indexPath: IndexPath) -> CGSize {
-        return CGSize(width: 320, height: 120)
+        let minimumWidth: CGFloat = 320
+
+        let availableWidth = self.collectionView!.frame.width - self.collectionView!.contentInset.left - self.collectionView!.contentInset.right
```

Comment:
I agree with you that forced unwrapping in general is not great, and would not be opposed to refactoring the file to avoid it.

I'm not going to change this usage for now to maintain consistency with the rest of the file (e.g. the existing `collectionView:cellForItemAt` below) and because we can't use the parameter instead of `self.collectionViewLayout` (because the type is not specific enough).